client/controllers/incidentReports/annotatedContent.coffee

```diff
@@ -27,7 +30,11 @@ Template.annotatedContent.onDestroyed ->
 Template.annotatedContent.helpers
   annotatedContent: ->
     selectedAnnotationId = Template.instance().data.selectedAnnotationId.get()
-    annotateContentWithIncidents(@content, @incidents.fetch(), selectedAnnotationId)
+    annotateContentWithIncidents(@content, @incidents.fetch().map((incident)->
+      if not incident.accepted
+        incident.uncertainCountType = true
```

Comment:
On the extract-incidents page uncertain incidents (where grits didn't think it was an incident) and rejected incidents (where the user removed an incident grits approved of) still have different highlights. The variable is necessary to maintain that functionality.